bundles/CoreBundle/DependencyInjection/Configuration.php

```diff
@@ -561,9 +566,11 @@ private function addAssetNode(ArrayNodeDefinition $rootNode)
                                         ])
                                     ->end()
                                     ->booleanNode('status_cache')
+                                        ->info("Store image metadata in a cache")
```

Comment:
As far as I checked this is more about a general performance improvement.
https://github.com/pimcore/pimcore/pull/11702 

As brusch mentions in his first comment:
This improves the performance significantly when using (remote) object storage for website projects.

Does this enhance your understanding on the config?